### PR TITLE
fix(sqs): fix unresolved promise blocking further serverless processes

### LINKED ISSUE #171: sls offline stuck on "offline: Starting Offline SQS: local/us-east-1."
Hi!
sls offline command stuck on "offline: Starting Offline SQS: local/us-east-1."

    "serverless-offline": "^6.8.0",
    "serverless-offline-sqs": "4.0.0", //  and the all newest

  serverless-offline-sqs:
    skipCacheInvalidation: false
    autoCreate: true
    endpoint: http://0.0.0.0:9324
    region: ${self:provider.region}
    accessKeyId: root
    secretAccessKey: root

What is the last working version of serverless-offline and serverless-offline-sqs ?

### PR DESCRIPTION
**Pull request by larrybotha**
fixes #171

### CODE REVIEW

File: packages/serverless-offline-sqs/src/index.js [hunk 1]
```diff
@@ -74,25 +74,27 @@ class ServerlessOfflineSQS {
...
-    });
+  _listenForTermination() {
+    const signals = ['SIGINT', 'SIGTERM'];
+
+    signals.map(signal =>
+      process.on(signal, async () => {
+        serverlessLog(`Got ${signal} signal. Offline Halting...`);
 
-    serverlessLog(`Got ${command} signal. Offline Halting...`);
+        await this.end();
```
Comment (dave-continuum-media) [inline packages/serverless-offline-sqs/src/index.js:90]: Does process.on callback ever support async functions?

Comment (larrybotha) [reply packages/serverless-offline-sqs/src/index.js:90]: Yup, run the script in the shell in the following repl.it: https://replit.com/@larrybotha1/processon-async#index.js

**Comment by bitsofinfo** [comment]
can this please be merged?  this is STILL an issue that is a giant time waster https://github.com/CoorpAcademy/serverless-plugins/issues/171

**Comment by paulcruse-syn** [comment]
I second that... can this be merged... adding `NODE_TEST` breaks other plugs (like [serverless-offline-dynamodb-local](https://www.serverless.com/plugins/serverless-dynamodb-local). 

This really causes issues, not sure how this even works for anyone at this point.

**Comment by HirenPatel2791** [comment]
Please merge this, still an issue in v6.0.0

**Comment by npthao1312** [comment]
Please merge this, this solves the problem for me.

**Comment by colesiegel** [comment]
Also looking to get this merged, plugin is unusable for us without it

**Comment by unrooty** [comment]
Bump! Looking forward to get this merged.

**Comment by godu** [comment]
@larrybotha Thank you for your work
https://github.com/CoorpAcademy/serverless-plugins/compare/148d096fb4a5685bb9f524f5e5d389bc640737a8...c42592a7a0e07141e702e1a78beb2acb41daa007
I applied the same logic to the others plugins. You will find it to the latest publish.

**Pull request closed without merging**